fastlane/Fastfile

```diff
@@ -97,402 +95,407 @@ lane :test do | options |
 end
 
 desc 'Builds the app using the App Store target but without code signing, just to check the target builds'
-lane :test_app_store_build do | options |
+lane :test_app_store_build do |_options|
   configure_apply
   build_simplenote(codesign: false)
 end
 
 ########################################################################
 # Release Lanes
 ########################################################################
-  #####################################################################################
-  # code_freeze
-  # -----------------------------------------------------------------------------------
-  # This lane executes the steps planned on code freeze
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane code_freeze [skip_confirm:<skip confirm>]
-  #
-  # Example:
-  # bundle exec fastlane code_freeze
-  # bundle exec fastlane code_freeze skip_confirm:true
-  #####################################################################################
-  desc "Creates a new release branch from the current trunk"
-  lane :code_freeze do | options |
-    old_version = ios_codefreeze_prechecks(options)
-
-    ios_bump_version_release(skip_deliver: true, skip_glotpress: true)
-    new_version = ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH)
-    ios_update_release_notes(new_version: new_version)
-    setbranchprotection(repository:GHHELPER_REPO, branch: "release/#{new_version}")
-    setfrozentag(repository:GHHELPER_REPO, milestone: new_version)
-
-    generate_strings_file_for_glotpress()
-  end
+#####################################################################################
+# code_freeze
+# -----------------------------------------------------------------------------------
+# This lane executes the steps planned on code freeze
+# -----------------------------------------------------------------------------------
+# Usage:
+# bundle exec fastlane code_freeze [skip_confirm:<skip confirm>]
+#
+# Example:
+# bundle exec fastlane code_freeze
+# bundle exec fastlane code_freeze skip_confirm:true
+#####################################################################################
+desc 'Creates a new release branch from the current trunk'
+lane :code_freeze do |options|
+  ios_codefreeze_prechecks(options)
+
+  ios_bump_version_release
+  new_version = ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH)
+  ios_update_release_notes(new_version: new_version)
+  setbranchprotection(
```

Comment:
💡  Same FYI as on iOS PR, this action has been renamed in release-toolkit `9.1.0` (though the old name is still around, just deprecated), so something to keep in mind for a future PR to address.

Reply:
Fixed in https://github.com/Automattic/simplenote-macos/commit/70698d2cfc07bd55d40a7060796518d08fb044b4